composer.json

```diff
@@ -18,7 +18,7 @@
         "bin/fqn-check"
     ],
     "require": {
-        "nikic/php-parser": "^3.0",
-        "symfony/finder": "^3.2"
+        "nikic/php-parser": "^4",
+        "symfony/finder": "^4"
```

Comment:
Why not `^3|^4` and `^3.2|^4`?

Reply:
Would be fine too, nevermind... just close that PR then and do it :-)